functions/_nvm_use_on_pwd_change.fish

```diff
@@ -0,0 +1,9 @@
+function _nvm_use_on_pwd_change \
+    --on-variable nvm_use_on_pwd_change \
+    --on-variable PWD \
+    --description 'Use Node.js version specified by project automatically'
+
+    if test "$nvm_use_on_pwd_change" = true
+        nvm use 2>/tmp/_nvm_use_on_pwd_change.log
```

Comment:
In my opinion, `nvm use` as it works currently isn’t ideal for automatic switching: https://github.com/jorgebucaran/nvm.fish/pull/186

Any thoughts?